.github/workflows/docker-supernode.yml

```diff
@@ -1,19 +1,19 @@
-name: Build docker client image
+name: Build docker SuperNode image
 
 on:
   workflow_dispatch:
     inputs:
       flwr-version:
-        description: "Version of Flower"
+        description: "Version of Flower."
         required: true
         type: string
 
 permissions:
   contents: read
 
 jobs:
-  build-client-images:
```

Comment:
As a potential follow up: We might want to combine this file with the SuperLink workflow to reduce the number of workflow files. wdyt?